tutorials/kafka-live-viewer-profiles/deploy-aws-terraform.md

```diff
@@ -56,24 +49,25 @@ $ ./down.sh  # <- stop the Docker containers
 $ ./up.sh    # <- start the Docker containers
 ```
 
-### Step 4: Open Access to the Applications
+## Step 4: Open access to the applications
 
-Review the [LocalStack guide](/tutorials/kafka-live-viewer-profiles/quickstart-localstack) for the default configuration for each component. Open public access to the two frontend applications and the Snowplow Collector using a HTTP load balancer so that anyone can watch the video, submit events to the pipeline, and see information on concurrent users.
+Review the [LocalStack guide](/tutorials/kafka-live-viewer-profiles/quickstart-localstack) for the default configuration for each component. Open public access to the two front-end applications and the Snowplow Collector using a HTTP load balancer so that anyone can watch the video, submit events to the pipeline, and see information on concurrent users.
 
 The applications listen for HTTP traffic on the following ports
-- Web tracker front end - 3000
-- Live viewer front end - 8280
-- Snowplow collector - 9090
+- Web tracker front-end - 3000
+- Live viewer front-end - 8280
+- Snowplow Collector - 9090
+
+## Next steps
 
-## Next Steps
 - You can implement Snowplow media tracking on any [HTML5](/docs/sources/trackers/javascript-trackers/web-tracker/tracking-events/media/html5/) or [YouTube](/docs/sources/trackers/javascript-trackers/web-tracker/tracking-events/media/youtube/) media of your choice
-- Look into the output from Kafka and extend the Live Viewer to include information on the media being watched and the user.
-- Replace Amazon DynamoDB with an alternative to be cloud agnostic, e.g. Google Bigtable or MongoDB.
+- Look into the output from Kafka and extend the Live Viewer to include information on the media being watched and the user
+- Replace Amazon DynamoDB with an alternative to be cloud agnostic, e.g. Google Bigtable or MongoDB
```

Comment:
Maybe use Redis here instead of Google Bigtable? I get the sentiment but Bigtable isn't cloud agnostic (GCP only).